docs/configuration/translation-strings.md

```diff
@@ -0,0 +1,62 @@
+# Translation Strings
+
+> Translation Strings are multilingual key-value pairs that you can use throughout the app. They enable you to translate
+> things like dropdown options, placeholder text, field notes, and more.
+
+[[toc]]
+
+## How It Works
+
+<video autoplay muted loop controls>
+<source src="https://cdn.directus.io/docs/v9/configuration/translation-strings/translation-strings-20220615A/translation-strings-20220615A.mp4" type="video/mp4" />
+</video>
+
+Translation Strings let you group together translations which display dynamically, whenever the app language is changed.
```

Comment:
I don't think this adds anything to the intro above. Let's just show the video and then dive into the headings. 👍